vignettes/estimate_ascertainment.Rmd

```diff
@@ -223,45 +216,18 @@ df_reporting_varying
 
 Finally, we estimate ascertainment for all countries with large early outbreaks of Covid-19.
 
-We define a large outbreak as one which has caused at least 100,000 deaths, and focus on the period between the start of each outbreak to the 1st June 2020.
+We define a large outbreak as one which has caused at least 100,000 deaths between 2020 and 2023, and focus on the period between the start of each outbreak to the 1st June 2020.
 
 We do so as it was this period where under-ascertainment of cases and infections was likely the highest, as tests were still being developed and had not been made widely available in many countries.
 
-We load the [global Covid-19 dataset collected by Our World in Data](https://ourworldindata.org/coronavirus), and provided through the [_owidR_ package](https://cran.r-project.org/package=owidR).
-
-```{r}
-# get data from {owidR}
-df_covid <- owid_covid()
-
-# filter out whole continents
-df_covid <- filter(
-  df_covid, location != continent
-)
-
-# rename the cases and deaths columns using {dplyr}
-df_covid <- select(
-  df_covid, iso_code, location, continent, date,
-  cases = new_cases, deaths = new_deaths,
-  total_deaths
-)
-```
+We now use the larger dataset `covid_data` made available with the _cfr_ package, and initially made available via the _covidregionaldata_ package [@palmer2021].
 
 We adopt data science tools to iteratively apply the `estimate_ascertainment()` function across data grouped by country.
 We refer the user to the book [R for Data Science](https://r4ds.had.co.nz/) for a better explanation of some of the code used here, including from the packages in [the Tidyverse](https://www.tidyverse.org/).
 
-
 ```{r}
-# calculate the total number deaths by country and filter for countries
-# with 100,000 deaths, and then filter for the first six months of 2020
-df_reporting <- df_covid %>%
-  group_by(iso_code) %>%
-  mutate(total_deaths = max(total_deaths, na.rm = TRUE)) %>%
-  filter(total_deaths > 100000 & !is.na(location)) %>%
-  filter(date < "2020-06-01") %>%
-  ungroup()
-
 # nest the data
-df_reporting <- nest(df_reporting, .by = location)
+df_reporting <- nest(covid_data, .by = country)
 
 # calculate the reporting rate in each country using
 # map on nested dataframes
```

Comment:
The `df_reporting` chunk returns a couple of errors for me:
```
Error in `mutate()`:
! Problem while computing `reporting = map(...)`.
Caused by error in `map()`:
! `.x` must be a vector, not a function.
```
and for the unnesting:
```
Error in `chr_as_locations()`:
! Can't subset columns that don't exist.
✖ Column `reporting` doesn't exist.
```
and hence for the plot:
```
Error in `geom_pointrange()`:
! Problem while computing aesthetics.
ℹ Error occurred in the 1st layer.
Caused by error in `check_factor()`:
! object 'country' not found
```